router/pump.go

```diff
@@ -140,7 +140,7 @@ func (p *LogsPump) Run() error {
 		debug("pump.Run() event:", normalID(event.ID), event.Status)
 		switch event.Status {
 		case "start", "restart":
-			go p.pumpLogs(event, true, inactivityTimeout)
+			go p.pumpLogs(event, false, inactivityTimeout)
```

Comment:
We can create an option then, if someone needs to re-send them.

Leaving it on `start` will not help as you can stop/start and it is all sent again.